CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
 - Support `on-demand-install-capable` application https://github.com/tuist/XcodeProj/pull/554 by @d-date
 - Add RemotePath to RemoteRunnable https://github.com/tuist/XcodeProj/pull/555 by @kwridan
 
+### Fixed
+- lastKnownFileType for .ttf and .sqlite files https://github.com/tuist/XcodeProj/pull/557 by @adamkhazi
```

Comment:
Can we move this into the Next section above

Reply:
Thanks, I've moved it!